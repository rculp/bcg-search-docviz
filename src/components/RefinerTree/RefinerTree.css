.refiner-tree {
  z-index: 0;
}
