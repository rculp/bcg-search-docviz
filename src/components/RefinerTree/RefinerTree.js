import React from 'react';

import { Header } from 'semantic-ui-react';

import './RefinerTree.css';

const RefinerTree = ({ box }) => (
  <div className="refiner-tree">
    <Header as="h2">{box.display}</Header>
    <div>Tree Refiner - Sinequa doesn&apos;t seem to return the actual items. Possibly need to make another API call?</div>
  </div>
);

export default RefinerTree;
