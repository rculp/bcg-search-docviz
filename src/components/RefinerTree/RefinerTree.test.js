import React from 'react';
import { shallow } from 'enzyme';

import RefinerTree from './RefinerTree';

describe('Refiner List', () => {
  const props = {
    box: {
      display: 'displayName'
    }
  };

  it('renders and matches our snapshot', () => {
    const component = shallow(
      <RefinerTree {...props} />
    );
    expect(component).toMatchSnapshot();
  });
});