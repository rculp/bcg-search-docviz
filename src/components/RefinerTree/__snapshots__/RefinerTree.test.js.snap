// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Refiner List renders and matches our snapshot 1`] = `
<div
  className="refiner-tree"
>
  <Header
    as="h2"
  >
    displayName
  </Header>
  <div>
    Tree Refiner - Sinequa doesn't seem to return the actual items. Possibly need to make another API call?
  </div>
</div>
`;
